pkg/mappings.bzl

```diff
@@ -537,6 +537,30 @@ pkg_mklink = rule(
     provides = [PackageSymlinkInfo],
 )
 
+def pkg_mklink(name, target, dest, attributes=None, src=None, **kwargs):
+  """Create a symlink.
+
+  Args:
+    name: target name
+    target: target path that the link should point to.
+    dest: the path in the package that should point to the target.
+    attributes: file attributes.
+  """
+  if src:
+    if target:
+      fail("You can not specify both target and source. Use source")
+    print("Warning: pkg_mklink.src is deprecated. Use target.")
```

Comment:
Added buildifer. 
There seems to be religion about never printing things except errors. I am an apostate that believes deprecation warnings are of value rather than noise. I gave up fighting the battle for support in bazel.